corehq/apps/es/es_query.py

```diff
@@ -394,6 +380,9 @@ def facets(self):
         results = namedtuple('facet_results', [f.name for f in facets])
         return results(**{f.name: f.parse_result(raw) for f in facets})
 
+    def __repr__(self):
+        return '{}({!r}, {!r})'.format(self.__class__.__name__, self.raw, self.query)
```

Comment:
It might be and it might not be depending on the query. I think if you're printing out the repr of an  `ESQuerySet` for debugging, one of the things you're likely interested in is the raw results. I'm inclined to leave like this until it annoys rather than helps someone, and then I certainly won't object to them changing it.